app/_components/form/CheckboxGroup.tsx

```diff
@@ -4,6 +4,7 @@ import { PropsWithChildren } from 'react';
 
 const CheckboxGroup = ({ children }: PropsWithChildren<{}>) => (
   <Stack
+    height={{ base: 'inherit', md: '40px', lg: '40px' }}
```

Comment:
The input needs to wrap on mobile

![Selection_024](https://github.com/ed-pilots-network/frontend/assets/67390534/66b575ee-91bb-4924-9fca-890b53732d44)